Meta/find_compiler.py

```diff
@@ -18,8 +18,8 @@
 from Meta.host_platform import Platform
 from Meta.utils import run_command
 
-CLANG_MINIMUM_VERSION = 19
-GCC_MINIMUM_VERSION = 14
+CLANG_MINIMUM_VERSION = 20
```

Comment:
? The documentation does not state we require clang 20, it just states that is what is used in CI.

Reply:
Then I cant read, will fix